spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -695,6 +695,20 @@ class Point < Struct.new :x, :y
       RUBY
     end
 
+    it 'register an offense in calls inside braced blocks' do
+      expect_offense(<<~RUBY)
+        client.images(page: page) { |resource| Image.new(resource) }
+                                                        ^^^^^^^^^^ Omit parentheses for method calls with arguments.
+      RUBY
```

Comment:
Can you add `expect_correction`?

```suggestion
      RUBY
      
      expect_correction(<<~RUBY)
        client.images(page: page) { |resource| Image.new resource }
      RUBY
```

Reply:
Sure, will add it.